py3status/modules/disk_free_space.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Display the amount of free space of given disk.
+This modules requires the psutil python module.
+
+Configuration parameters:
+    path: filesystem path to check
+        (default: "/")
+    threshold_degraded / threshold_bad: thresholds for color change
+    format: format string
+        (default: "{space:.1f}G")
+
+Format string parameters:
+    {space} space left (in GB)
+    {path} path checked
+"""
+
+# import your useful libs here
+import psutil
```

Comment:
Is there a reason you are using psutil?  I've got a disk space module that uses `df` to get the info.